.github/workflows/reusable_checks.yml

```diff
@@ -266,6 +266,7 @@ jobs:
       - name: Install dependencies
         run: |
           pip install gitignore_parser python-frontmatter
+          pip install -r ./scripts/ci/requirements.txt
```

Comment:
Neither of these should be needed if we would run the job on our `ci_docker` container

```
    container:
      image: rerunio/ci_docker:0.9
```

Reply:
I think the docker container is a hack we should use as little as possible, because someone always has to remember to update it any time we add new dependencies. The job here takes 1 minute to run, and it runs in parallel to all the other checks.

Reply:
I wanted to try it to see how much of a speedup it would be, because I suspect it might actually be a slowdown, but:

> python3: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.34' not found (required by python3)

Now I don't want to touch it in fear of what other horrors lie beyond. 🙂